Example/EasyTransitions/ViewControllers/Today/TodayCollectionViewController.swift

```diff
@@ -32,9 +40,30 @@ class TodayCollectionViewController: UICollectionViewController {
         collectionView?.register(TodayCollectionViewCell.self)
     }
 
+    override func viewWillLayoutSubviews() {
```

Comment:
I'd prefer we invalidate the layout of the `collectionView` on `viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator)` so it we make sure to force the layout change only when the user is rotating.

Reply:
That’s an interesting alternative! I’ll make the change.